modules/spark/src/main/scala_2.11/spark-last/notebook/kernel/Repl.scala

```diff
@@ -103,11 +103,13 @@ class Repl(val compilerOpts: List[String], val jars:List[String]=Nil) extends Re
     //val i = new HackIMain(settings, stdout)
     loop = new org.apache.spark.repl.HackSparkILoop(stdout, outputDir)
 
-    jars.foreach { jar =>
+    val fps = jars.map{ jar =>
       import scala.tools.nsc.util.ClassPath
       val f = scala.tools.nsc.io.File(jar).normalize
-      loop.addedClasspath = ClassPath.join(loop.addedClasspath, f.path)
+      //loop.addedClasspath = ClassPath.join(loop.addedClasspath, f.path)
+      f.path
     }
+    settings.classpath.value=fps.mkString(java.io.File.pathSeparator)
```

Comment:
is it OK that this is overriding and loosing some older values of `settings.classpath.value`?
see https://github.com/spark-notebook/spark-notebook/pull/964/files#diff-42e4d431da236e133a4c232a417ac431R94

Reply:
Good point!!
I guess we could do : 
`settings.classpath.value=(classpath.distinct ::: fps).mkString(java.io.File.pathSeparator)`

and remove that other line?

Reply:
sounds legit